core/shared/src/main/scala/scalaz/zio/UIO.scala

```diff
@@ -0,0 +1,285 @@
+
+  /** See [[scalaz.zio.ZIO.identity]]
+   */
+  final def identity: UIO[Any] = ZIO.identity
+
+  /** See [[scalaz.zio.ZIO.interruptible]]
+   */
+  final def interruptible[A](zio: UIO[A]): UIO[A] =
+    ZIO.interruptible(zio)
+
+  /** See [[scalaz.zio.ZIO.interruptibleMask]]
+   */
+  final def interruptibleMask[A](k: ZIO.InterruptStatusRestore => UIO[A]): UIO[A] =
+    ZIO.interruptibleMask(k)
+
+  /** See [[scalaz.zio.ZIO.lock]]
+   */
+  final def lock[A](executor: Executor)(zio: UIO[A]): UIO[A] =
+    ZIO.lock(executor)(zio)
+
+  /** See [[scalaz.zio.ZIO.mergeAll]]
+   */
+  final def mergeAll[A, B](in: Iterable[UIO[A]])(zero: B)(f: (B, A) => B): UIO[B] =
+    ZIO.mergeAll(in)(zero)(f)
+
+  /** See [[scalaz.zio.ZIO.mergeAllPar]]
+   */
+  final def mergeAllPar[A, B](in: Iterable[UIO[A]])(zero: B)(f: (B, A) => B): UIO[B] =
+    ZIO.mergeAllPar(in)(zero)(f)
+
+  /** See [[scalaz.zio.ZIO.raceAll]]
+   */
+  final def raceAll[R1 <: Any, A](zio: UIO[A], ios: Iterable[ZIO[R1, Nothing, A]]): ZIO[R1, Nothing, A] =
+    ZIO.raceAll(zio, ios)
+
+  /** See [[scalaz.zio.ZIO.reduceAll]]
+   */
+  final def reduceAll[R1 <: Any, A](a: UIO[A], as: Iterable[ZIO[R1, Nothing, A]])(f: (A, A) => A): ZIO[R1, Nothing, A] =
+    ZIO.reduceAll(a, as)(f)
+
+  /** See [[scalaz.zio.ZIO.reduceAllPar]]
+   */
+  final def reduceAllPar[R1 <: Any, A](a: UIO[A], as: Iterable[ZIO[R1, Nothing, A]])(
+    f: (A, A) => A
+  ): ZIO[R1, Nothing, A] =
+    ZIO.reduceAllPar(a, as)(f)
+
+  /** See [[scalaz.zio.ZIO.reserve]]
+   */
+  def reserve[A, B](reservation: UIO[Reservation[Any, Nothing, A]])(use: A => UIO[B]): UIO[B] =
+    ZIO.reserve(reservation)(use)
+
+  /** See [[scalaz.zio.ZIO.runtime]]
+   */
+  final def runtime: UIO[Runtime[Any]] = ZIO.runtime
+
+  /** See [[scalaz.zio.ZIO.succeed]]
+   */
+  final def succeed[A](a: A): UIO[A] = ZIO.succeed(a)
+
+  /** See [[scalaz.zio.ZIO.succeedLazy]]
+   */
+  final def succeedLazy[A](a: => A): UIO[A] = ZIO.succeedLazy(a)
+
+  /** See [[scalaz.zio.ZIO.supervise]]
+   */
+  final def supervise[A](zio: UIO[A]): UIO[A] =
+    ZIO.supervise(zio)
+
+  /** See [[scalaz.zio.ZIO.supervised]]
+   */
+  final def supervised[A](zio: UIO[A]): UIO[A] =
+    ZIO.supervised(zio)
+
+  /** See [[scalaz.zio.ZIO.superviseWith]]
+   */
+  final def superviseWith[A](zio: UIO[A])(supervisor: IndexedSeq[Fiber[_, _]] => UIO[_]): UIO[A] =
+    ZIO.superviseWith(zio)(supervisor)
+
+  /** See [[scalaz.zio.ZIO.suspend]]
+   */
+  final def suspend[A](io: => UIO[A]): UIO[A] =
+    ZIO.suspend(io)
+
+  /** See [[scalaz.zio.ZIO.interruptibleMask]]
+   */
+  final def uninterruptible[A](zio: UIO[A]): UIO[A] =
+    ZIO.uninterruptible(zio)
+
+  /** See [[scalaz.zio.ZIO.uninterruptibleMask]]
+   */
+  final def uninterruptibleMask[A](k: ZIO.InterruptStatusRestore => UIO[A]): UIO[A] =
+    ZIO.uninterruptibleMask(k)
+
+  /** See [[scalaz.zio.ZIO.unsandbox]]
+   */
+  final def unsandbox[A](v: IO[Cause[Nothing], A]): UIO[A] = ZIO.unsandbox(v)
```

Comment:
I'd delete `unsandbox` from `UIO` because the input to this function cannot be typed as `UIO`.